task/07-yield-tasks.js

```diff
@@ -126,7 +164,21 @@ function* breadthTraversalTree(root) {
  *   [ 1, 3, 5, ... ], [ -1 ] => [ -1, 1, 3, 5, ...]
  */
 function* mergeSortedSequences(source1, source2) {
-    throw new Error('Not implemented');
+    let s1 = source1(),
+    s2 = source2(),
+    value1 = s1.next().value,
+    value2 = s2.next().value;
+    while (value1 !== undefined || value2 !== undefined) {
```

Comment:
this is not going to work for the case when you need to return several values from the source1 in a row